build.gradle

```diff
@@ -122,6 +122,18 @@ subprojects {
           name = "projectLocalMaven"
           url = "${rootProject.buildDir}/localMaven"
         }
+        /**
+         * Are you a Square employee?  If so, then set the following properties in
+         * ~/.gradle/gradle.properties to enable pushing to our internal repository
+         *
+         * artifactoryUsername=YOUR_LDAP
+         * artifactoryPassword=YOUR_API_KEY
+         */
+        maven {
+          name = "artifactory"
+          url = "https://maven.global.square/artifactory/releases"
```

Comment:
We're not supposed to leak internal URLs. Can you use an environment variable? For Treehouse we use `-PRELEASE_REPOSITORY_URL="the url"` on the CLI to release internally.

Reply:
ok, addressed by generalizing to any internal repository.  PTAL!